scripts/build_addon.sh

```diff
@@ -0,0 +1,64 @@
+# !/bin/bash
+
+if [ "$#" -eq 0 ]; then
+	echo "usage build_addon <product path> <release/debug - optional>"
+	exit
+fi
+
+PRODUCT_ROOT_PATH=$1
+BUILD_CONF=
+ADDON_PATH=$PRODUCT_ROOT_PATH/node_addons/FormatConverter
+FORMAT_CONVERTER_BIN=FormatConverter.so
+OS=`uname`
+echo "OS=$OS"
+
+[ "$#" -eq 2 ] && BUILD_CONF=$2 || BUILD_CONF=Release
+
+if [ "$BUILD_CONF" = "debug" ]; then
```

Comment:
Why not just say you expect 'Release' || 'Debug' in the first place? rather than expect and check for lowercase only to ucf() it?